resources/views/admin/scholarship/partials/_form_scholarship.blade.php

```diff
@@ -157,8 +164,8 @@
 
 {{-- Recommendation Optional question --}}
 <div class="form-group">
-  {!! Form::label('display_optional_rec_question', 'Display optional recommendation question? ') !!}
-  {!! Form::checkbox('display_optional_rec_question', 1, ['class' => 'form-control']) !!}
+  {!! Form::label('display_optional_rec_question', 'Display optional recommendation question?') !!}
+  {!! Form::checkbox('display_optional_rec_question', 1, $scholarship->display_optional_rec_question) !!}
```

Comment:
Do you not need to pass the class anymore?

Reply:
@weerd for some reason it started to look weird with it, so I removed it and it looks fine now

Reply:
![image](https://user-images.githubusercontent.com/4240292/29933588-2abb232a-8e46-11e7-9c9a-219b27145fb0.png)